src/realm/group_writer.cpp

```diff
@@ -524,9 +521,14 @@ ref_type GroupWriter::write_group()
     top.set(Group::s_version_ndx, RefOrTagged::make_tagged(m_current_version)); // Throws
     if (m_evacuation_limit == 0) {
         size_t used_space = m_logical_size - m_free_space_size;
-        if (m_logical_size >= 0x100000 && m_free_space_size - m_locked_space_size > 2 * used_space) {
+        // Compacting files smaller than 1 Mb is not worth the effort. Arbitrary chosen value.
+        static constexpr size_t one_mb = 0x100000;
```

Comment:
this is nitpicking, but wouldn't it be better if the named constants had names which reflected their use/meaning instead of just being textual versions of their values? (sixty_four_kb could be minimal_evac_limit, for example)

Reply:
Yes, I think so too. I don't know why, but I was somehow of mislead bu your previous comment. I'll change.

Reply:
I hereby take full responsibility! :-D